docs/3.1-Shadow-Config.md

```diff
@@ -140,24 +140,24 @@ Type: String OR null
 
 Path to recursively copy during startup and use as the data-directory.
 
-#### `topology`
+#### `network`
 
 *Required*
 
 Network topology settings.
 
-#### `topology.graph`
+#### `network.graph`
 
 *Required*
 
 The network topology graph.
 
-The topology, a collection of nodes and edges that represent a network topology, should hold an undirected, connected graph with certain attributes specified on the nodes and edges. For more information on how to structure this data, see the [Topology Format](3.2-Network-Config.md).
+A collection of nodes and edges that represent a network topology, should hold an undirected, connected graph with certain attributes specified on the nodes and edges. For more information on how to structure this data, see the [Topology Format](3.2-Network-Config.md).
```

Comment:
> should hold an undirected, connected graph

The graph can be directed or undirected.

Reply:
Fixed!